frontend-public/src/utils/customAxios.js

```diff
@@ -5,6 +5,10 @@ import * as Strings from "@/constants/strings";
 const UNAUTHORIZED = 401;
 const MAINTENANCE = 503;
 
+// https://stackoverflow.com/questions/39696007/axios-with-promise-prototype-finally-doesnt-work
+const promiseFinally = require("promise.prototype.finally");
+promiseFinally.shim();
```

Comment:
Why do we need this? The link is from 2016. The first example in Axios v0.19 docs use `finally`, so it should be fully-supported

Reply:
Without this all of the unit tests failed for all of the action creators. Maybe we can raise this as a separate issue?

Reply:
Oh, weird... Yeah, it can likely just be a separate issue. I don't think it's worth it to block the rest of these changes